src/snek5000/output/base.py

```diff
@@ -298,7 +290,7 @@ def __init__(self, sim=None, params=None):
         else:
             self.package = get_solver_package(self.name_solver)
 
-        self.root = self.get_root()
+        self.path_solver_package = self.get_path_solver_package()
```

Comment:
Add a method `get_root`, which issues `DeprecationWarning` and returns `get_path_solver_package` instead. That will make merging this PR smoother.

Reply:
Done in a9cb307f7dc013407bc4e695ecd6d9cd690f1b33